docs/tutorials/json_schema.md

```diff
@@ -0,0 +1,163 @@
+In this tutorial, you will learn how to bind a JSON Schema to a folder. You can also bind a JSON Schema to a project, a file, a table, or a view. This tutorial will walk you through each step, from schema creation to validation.
+
+## Tutorial Purpose
+By the end of this tutorial, you will:
+
+1. Log in and create a project and folder
+2. Create a JSON Schema and an organization
+3. Bind the JSON Schema to a folder
+4. Add annotations and validate against the schema
+5. Attach and validate a file
+6. View schema validation statistics and results
+
+## Prerequisites
+* You have a working installation of the Synapse Python Client
+* You have a project on Synapse ready for testing
+* You are familiar with Synapse concepts: Project, Folder, File
+* You are authenticated `(syn = synapseclient.Synapse(); syn.login())`
+
+
+##  Step 1: Define a JSON Schema
+```python
+title = "OOP Test Schema"
```

Comment:
Rather than duplicating the python code the preferences instead is to tell mkdocs to grab certain lines from the python script. See other markdowns for examples